pyproject.toml

```
@@ -12,7 +12,7 @@ readme = "README.md"
 license = "GPL-3.0-or-later"
 dependencies = [
     "python-dateutil==2.9.0.post0",
-    "jsonschema[format]==4.24.0",
+    "jsonschema[format]==4.25.0",
```

@CBeck-96 can you please make the change

```suggestion
    "jsonschema[format-nongpl]==4.25.0",
```